packages/contracts-bedrock/src/universal/FeeVault.sol

```diff
@@ -17,20 +18,32 @@ abstract contract FeeVault {
     }
 
     /// @notice Minimum balance before a withdrawal can be triggered.
+    ///         Use the `minWithdrawalAmount()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     uint256 public immutable MIN_WITHDRAWAL_AMOUNT;
 
-    /// @notice Wallet that will receive the fees.
+    /// @notice Account that will receive the fees. Can be located on L1 or L2.
+    ///         Use the `recipient()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     address public immutable RECIPIENT;
 
-    /// @notice Network which the RECIPIENT will receive fees on.
+    /// @notice Network which the recipient will receive fees on.
+    ///         Use the `withdrawalNetwork()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     WithdrawalNetwork public immutable WITHDRAWAL_NETWORK;
 
     /// @notice The minimum gas limit for the FeeVault withdrawal transaction.
-    uint32 internal constant WITHDRAWAL_MIN_GAS = 35_000;
+    uint32 internal constant WITHDRAWAL_MIN_GAS = 400_000;
```

Comment:
Why was this increased?

Reply:
one possibility is because `SafeCall.send(address,uint256)` is now being used instead of a standard Solidity call: https://github.com/ethereum-optimism/optimism/pull/10685/files/b039194caf99b8822bb471184840c58ae48a21af#diff-020187e3fd9fa7ac087a484cb07894f7c2365c5cc0fad519f77ca4ac03c8ed96L74-R102

`SafeCall`'s `send` method likely consumes more gas than simple call, explaining the increase in `WITHDRAWAL_MIN_GAS` (which must cover the cost of the call)

Reply:
ill let @tynes confirm before resolving

Reply:
The `WITHDRAWAL_MIN_GAS` is increased so that the withdrawal call coming out of the `OptimismPortal` has more gas supplied to it. The value is passed to `L2ToL1MessagePasser.initiateWithdrawal` which commits to the call that comes out of the `OptimismPortal` on L1

This will guarantee that the withdrawal succeeds on L1 with a large buffer. Longer term we should really remove the concept of gas limits on cross chain messages as it results in bad ux, or if we want to keep it we should add replayability to the portal itself to remove this sort of bug